frontend/src/styles/_card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  $root: &;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  min-height: 57.1rem;
+
+  @media (max-width: 1919px) {
+    min-height: 44.5rem;
+  }
+
+  @media (max-width: 1439px) {
+    min-height: 32.5rem;
+  }
+
+  &__img {
+    background: #DBDCDD;
+    width: 100%;
+    height: 100%;
+    border-radius: 1.2rem;
+  }
```

вот сюда добавить стиль
```suggestion
  &__img {
    background: #DBDCDD;
    width: 100%;
    height: 100%;
    border-radius: 1.2rem;
    aspect-ratio: 116/167;
  }
```